gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -46,17 +47,19 @@ public String getLastActiveProjectId() {
 
   /** Set the id of the last project successfully activated by the user. */
   public void setLastActiveProjectId(@NonNull String id) {
-    preferences
-      .edit()
-      .putString(ACTIVE_PROJECT_ID_KEY, id)
-      .apply();
+    preferences.edit().putString(ACTIVE_PROJECT_ID_KEY, id).apply();
   }
 
   /** Removes the last active project id in the local value store. */
   public void clearLastActiveProjectId() {
-    preferences
-      .edit()
-      .remove(ACTIVE_PROJECT_ID_KEY)
-      .apply();
+    preferences.edit().remove(ACTIVE_PROJECT_ID_KEY).apply();
+  }
+
+  public boolean isOfflineModeEnabled() {
```

Comment:
Is enabling offline mode explicitly part of the intended design? On the offline imagery side of things, I believe our decision was to render offline imagery automatically, without the user having to specify any sort of offline mode as off or on. I might be mistaken about this, though.

It seems odd to me to maintain to separate approaches to our handling of the offline modality. 

@gino-m

Reply:
So sorry for the confusion.

No, it's not a part of the design. I added it as a debug-only feature while working on the issue. It's easier to test the offline behavior for a specific target without cutting-off the network connectivity for all other elements (which still rely on internet!)

My plan is to remove this added UI feature and replace the usages of `ifOfflineModeEnabled()` with `isOffline()` later, i.e. when all modules support offline mode.

Reply:
Correct. The user is offline implicitly when they don't have a network connection. Airplane mode can be a proxy for this, so we don't need a UI to force them offline.

Reply:
Afaik, Project and related entities like Layer and Form are the last ones that aren't yet serialized locally for offline use. In that case would it make sense to remove it now?